application/src/main/java/com/sanction/lightning/FacebookProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.sanction.lightning;
+
+import com.restfb.DefaultFacebookClient;
+import com.restfb.Parameter;
+import com.restfb.Version;
+import com.sanction.lightning.authentication.FacebookAuth;
+import com.sanction.lightning.models.FacebookUser;
+import com.sanction.thunder.models.StormUser;
+
+public class FacebookProvider {
+
+  StormUser user;
+  DefaultFacebookClient client;
+
+  /**
+   * Constructs a new FacebookProvider for making requests to facebook.
+   * @param user
+   */
+  public FacebookProvider(StormUser user, FacebookAuth auth) {
```

Comment:
Maybe instead of taking in a `StormUser` you take in `String userAcessToken`. You don't really need the entire user. And you wouldn't even need to save the access token as a class variable.